BigBaseV2/src/pointers.cpp

```diff
@@ -395,13 +393,38 @@ namespace big
 			m_online_version = ptr.add(0x24).rip().add(0x20).as<const char*>();
 		});
 
+		// Send Metric a
+		main_batch.add("SMA", "48 89 5C 24 08 57 48 83 EC 20 48 8B D9 33 C9 48 8B FA E8 ?? ?? ?? ?? 48", [this](memory::handle ptr)
```

Comment:
Invalid signature, uses double ?? IDA signature instead of the single ? IDA signature.